doc/src/vpr/file_formats.rst

```diff
@@ -411,15 +411,13 @@ For example:
     .param multiplier 0.50
     .param power 001101
 
-Would set the parameters ``feedback``, ``multiplier`` and ``power``  of the above ``pll`` ``.subckt`` to ``"internal"``, ``0.50`` and ``001101`` respectively.
-
-.. warning:: Integers in notation other than binary (e.g. decimal, hexadecimal) are not supported. Occurrence of params with digits other than 1 and 0 for binary words, not quoted (strings) or not separated with dot ``.`` (real numbers) are considered to be illegal.
```

Comment:
Did you add support for decimal and hexa? otherwise the warning should be kept in the docs.

Reply:
Actually, I did not make these changes, so I don't know whether the support is added. It seems that @lpawelcz and @mkurc-ant accidentally change my branch. If you think it should be reversed, I will do that.